src/components/PersonList.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+interface Props {
+  preparedPeopleList: Person[];
+  handleSelectPerson: (person: Person) => void;
+}
+
+export const PersonList: React.FC<Props> = ({
+  preparedPeopleList,
+  handleSelectPerson,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      {preparedPeopleList.length ? (
+        <div
+          className="dropdown-content"
+          style={{
+            maxHeight: 400,
+            overflowY: 'auto',
+          }}
+        >
+          {preparedPeopleList.map(person => (
+            <div
+              className="dropdown-item"
+              key={person.slug}
+              onKeyDown={() => { }}
```

Comment:
Why do we need to return the object?

Reply:
we should write function here, and it's anonymous function